mbed_greentea/mbed_greentea_cli.py

```diff
@@ -696,6 +696,7 @@ def get_parallel_value(value):
 
     # Detect devices connected to system
     mbeds = mbed_lstools.create()
+    mbeds.mbedls_get_mocks()
```

Comment:
That makes sense to me. It was removed from the constructor of mbed-ls here: https://github.com/ARMmbed/mbed-ls/commit/1e48f7ad8386e50cf58307bb36748ba99185bc0c

Though it looks like it wasn't included back in to the flow when it was broken out into a separate function, so perhaps this was a mistake?